REST client/trainings/src/main/java/training/evaluation/training/rest/TrainingResource.java

```diff
@@ -0,0 +1,24 @@
+package training.evaluation.training.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import training.evaluation.training.repository.TrainingRepository;
+import training.evaluation.training.model.Training;
+
+import javax.validation.Valid;
+
+@RestController
```

Comment:
Now, when I am trying to use  this endpoint, I think that the better approach is to set this annotation mapping.
@RequestMapping("/api/training")